apollo-federation/src/connectors/json_selection/README.md

```diff
@@ -91,7 +91,7 @@ VarPath              ::= "$" (NO_SPACE Identifier)? PathStep*
 KeyPath              ::= Key PathStep*
 AtPath               ::= "@" PathStep*
 ExprPath             ::= "$(" LitExpr ")" PathStep*
-PathStep             ::= "." Key | "->" Identifier MethodArgs?
+PathStep             ::= "." Key | "->" Identifier MethodArgs? | "?"
```

Comment:
The grammar will change slightly in #7993, but this version is not wrong (that is, it won't forbid any usages we want to support, though it might allow some we'd like to forbid, like multiple `missing???` operators).